models/im.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import "time"
+
+type Room struct {
+	ID        string    `json:"_id"`
+	UpdatedAt time.Time `json:"_updatedAt"`
+	Type      string    `json:"t"`
+	Messages  int       `json:"msgs"`
+	Timestamp time.Time `json:"ts"`
+	Meta      struct {
+		Revision int   `json:"revision"`
+		Created  int64 `json:"created"`
+		Version  int   `json:"version"`
+	} `json:"meta"`
+	Loki      int      `json:"$loki"`
```

Comment:
I think this should be gone.  It might used to have been used.  But loki internally should be gone.